charts/vsphere-cpi/v1.0.0/README.md

```diff
@@ -0,0 +1,33 @@
+# vSphere Cloud Provider Interface (CPI)
+
+[vSphere Cloud Provider Interface (CPI)](https://github.com/kubernetes/cloud-provider-vsphere) is responsible for running all the platform specific control loops that were previously run in core Kubernetes components like the KCM and the kubelet, but have been moved out-of-tree to allow cloud and infrastructure providers to implement integrations that can be developed, built and released independent of Kubernetes core. The official documentation and tutorials can be found [here](https://vsphere-csi-driver.sigs.k8s.io/driver-deployment/prerequisites.html).
+
+## Prerequisites
+
+- vSphere 6.7U3+
+- Secret with vSphere credentials
+
+## Installation
+
+1. Before installing the chart, you need to create a secret in the chart's namespace with the server URL and credentials to connect to the vCenter. To create a secret in Rancher, go to your cluster's project (Same project you will be installing the chart) > Resources > Secrets > Add Secret
```

Comment:
two comments:
1) we should just provide instructions to create this via kubectl instead of focusing on Rancher, e.g. creating the files and running `kubectl create secret generic <name-of-secret> -n <namespace> --from-file=<file> --from-file=<file>...` (as described in https://kubernetes.io/docs/tasks/configmap-secret/managing-secret-using-kubectl/)
2) If we can modify this helm chart, we should update it to either take in an existing secret name *or* create a secret with fields provided from the values.yaml that is managed via Helm to avoid forcing a user to create a secret outside the Helm chart if that's not what is required.

The prior can be modified in this PR, the latter can be done in a separate PR if we don't want to make those changes at the moment.